tl-tl/sc/white-paper.md

```diff
@@ -64,55 +64,56 @@ Ang environment ng pagpapatupad ng isang matalinong kontrata ay naglalaro ng isa
 
-NEO uses a lightweight NeoVM (NEO Virtual Machine) as its smart contract execution environment, which has a very fast start up and takes up very little resources, perfect for short programs like smart contracts. Using the compilation and caching of hotspot smart contracts with JIT (real-time compiler) can significantly improve the efficiency of virtual machines.
+Ang NEO ay gumagamit ng isang magaang NeoVM (Birtwal na Makina ng NEO) bilang matalinong kontratang execution environment nito, na mayroong isang napakabilis na start up at kumukuha ng napakaliit na mga resource, perpekto para sa maikling mga program katulad ng matalinong mga kontrata. Ang paggamit ng pag-compile at pag-cache ng hotspot na matalinong mga kontrata gamit ang JIT (totoong-oras na compiler) ay maaaring makabuluhang magpabuti sa kahusayan ng mga birtwal na makina.
 
-### 2.3 Scalability
+### 2.3 Kakayahan sa Pag-iskala
 
-#### 2.3.1 High concurrency and dynamic partitioning
+#### 2.3.1 Mataas na pagkakasabay at dynamic na pagpartisyon
 
-When discussing the scalability of a system, it involves two main areas: Vertical scaling and horizontal scaling. Vertical scaling refers to the optimization of the processing workflow, allowing the system to take full advantage of existing equipment capacity. With this approach, limits of the system are easily reached, as series-based processing capacity is based on the hardware limit of a single device. When we need to scale the system, is there a way to transform the series system into a parallel system? Theoretically, we will only need to increase the number of devices, and we will be able to achieve almost unlimited scalability. Could we possibly achieve unlimited scaling in distributed blockchain networks? In other words, can the blockchain execute programs in parallel?
+Kapag tayo ay nagtatalakay ng kakayahan sa pag-iskala ng isang sistema, ito ay nagsasangkot ng dalawang pangunahing mga lawak: Ang patayo na pag-iskala at pahalang na pag-iskala. Ang patayo na pag-iskala ay tumutukoy sa optimisasyon ng nagpoprosesong workflow, na nagpapahintulot sa sistema na magsamantala sa umiiral na kapasidad ng kagamitan. Sa diskarteng ito, ang mga limitasyon ng sistema ay madaling naabot, dahil ang series-based processing na kapasidad ay nakabase sa limitasyon ng hardware sa isang solong device. Kapag kailangan nating iiskala ang sistema, mayroon bang paraan upang mag-transform ng mga serye na sistema sa isang kahilera na sistema? Sa pagkateoretikal, kailangan lamang nating dagdagan ang bilang ng mga device, at maaari na nating makamit ang halos walang limitasyon na kakayahan sa pag-iskala. Pwede ba nating makamit ang walang limitasyon na pag-iskala sa ibinahaging mga blockchain network? Sa ibang salita, maaari bang magsagawa ang blockchain ng mga program na nakahilera?
```

Comment:
I would translate `areas/lawak` to `aspect/aspeto` or `field/saklaw` in tagalog.

Reply:
I would use `bertikal` instead of `patayo`

Reply:
`take full advantage of` / `magsamantala` to `gamitin ng husto`

Reply:
`diskarteng` to `pamamaraang`

Reply:
I would change `madaling naabot` to `madaling maaabot` since `easily reached` is in present perfect tense.

Reply:
`device` to `aparato`

Reply:
I have considered all your suggestions. Thank you.